raddb/mods-available/smtp

```diff
@@ -0,0 +1,275 @@
+		#
+		#  May be one of:
+		#
+		#  [options="header,autowidth"]
+		#  |===
+		#  | Option | Description
+		#  | `no`   | Server certificate CN can be any value.
+		#  | `yes`  | Server certificate CN matches the host in the URI.
+		#  |===
+		#
+		#  Default is `yes`
+		#
+#		check_cert_cn = no
+
+		#
+		#  extract_cert_attrs:: Extract OIDs from presented certificates as OIDs.
+		#
+		#  Default is `no`
+		#
+#		extract_cert_attrs = no
+	}
+
+	#
+	#  uri:: URI which will be used for connecting to the SMTP server.
+	#
+	#  The `smtp` module uses Curl (https://curl.haxx.se/libcurl/) to implement
+	#  the underlying mail protocols.  The URI should be in the form of:
+	#
+	#  `SCHEME://IP:PORT/`
+	#
+
+	#	`SCHEME` can be one of the protocols supported by Curl.
+	#	i.e. ftp, ftps, gopher, http, https, imap, imaps, ldap, ldaps,
+	#	     pop3, pop3s, rtsp, scp, sftp, smb, smbs, smtp, smtps.
+	#	If no `SCHEME` is given, it defaults to `smtp`.
+	#
+	#	`IP` is the IP address of the server.  It can be an IPv4 address,
+	#	IPv6 address, hostname, etc.
+	#
+	#	`PORT` is optional, and will normally be chosen to be correct
+	#	for the given `SCHEME`.
+	#
+	#  For more information, see the CURL documentation at
+	#
+	#	https://ec.haxx.se/cmdline/cmdline-urls
+	#
+	uri = "smtp://192.0.20.1/"
+
+	#
+	#  timeout:: How long the module will wait before giving up on the response
+	#  from the SMTP server.
+	#
+	timeout = 5s
+
+	#
+	#  template_directory:: The source directory where all file attachments are pulled from
+	#  All file attachments should be their relative path from this location, without a leading /
+	#
+	template_directory = raddb/mods_config/smtp
+
+	#
+	#  envelope_address:: This is the address used to send the mail,
+	#  Sent to the receiving server as MAIL FROM:<envelope_address>
+	#  sender_address will be defaulted to if this is not set
+	#
+	envelope_address = &SMTP-Sender-Address
+
+	#
+	#  sender_address:: This is the address displayed in the FROM: element of the header
+	#  This can be different than the provided envelope_address,
+	#  If envelope_address is set, this can be formatted however you want it to appear to the receiver
+	#
+	sender_address = &SMTP-Sender-Address
+
+	#
+	#  recipients:: Comma separated list of emails used to generate the envelope recipients
+	#  the local part may contain commas, the domain may not (RFC 2821)
+	#  Therefore, a comma after the @ represents a new address
+	#  If any recipients are set; to, cc, bcc are only used to generate the headers
+	#  Additional recipients can be added by updating SMTP-Recipients
+	#
+#	recipients = "ex@ample.com,ex2@ample.com"
+
+	#
+	#  to:: Comma separated list of emails.
+	#  the local part may contain commas, the domain may not (RFC 2821)
+	#  Therefore, a comma after the @ represents a new address
+	#  listed in the TO: header of the email
+	#  if recipients is not set, these emails will be added to the envelope recipients
+	#
+#	to = "ex@ample.com,ex2@ample.com"
```

Comment:
Shouldn't be comma separated.  You're introducing domain specific formatting for emails into the FreeRADIUS configuration scheme.  Use `FR_TYPE_MULTI` for the config item, and build the list of emails from the tmpls you get.